swagger_templates/python/model.mustache

```diff
@@ -75,35 +75,6 @@ class {{classname}}(object):
         :type: {{datatype}}
         """
         {{#isEnum}}allowed_values = [{{#allowableValues}}{{#values}}"{{{this}}}"{{^-last}}, {{/-last}}{{/values}}{{/allowableValues}}]
-{{#isContainer}}
```

Comment:
Again, these changes were introduced when I upgraded our original template to the latest version of the template from the upstream Swagger. So now they are being removed by moving back to the version of the template that comes with the version of Swagger Codegen that we use for our build.